Step/Type/PaymentStepType.php

```diff
@@ -66,24 +74,27 @@ public function configureOptions(OptionsResolver $resolver)
 
         $resolver
             ->setRequired([
-                'amount',
-                'currency_code',
-                'item_id',
                 'payment_gateway_configuration_alias',
             ])
             ->setDefaults([
+                'amount' => null,
+                'item_id' => null,
+                'currency_code' => null,
                 'callback_url' => null,
                 'customer_email' => null,
                 'customer_id' => null,
                 'description' => null,
                 'return_url' => null,
             ])
             ->setNormalizer('amount', function (Options $options, $value) {
-                return intval($value, 10);
+                return null !== $value ? intval($value, 10) : -1;
```

Comment:
keep null value

Reply:
Fixed in 88f667bc281e2a0b047e19831679f7e71c35441f